go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	cloud.google.com/go/kms v1.1.0
 	cloud.google.com/go/pubsub v1.16.0
-	cloud.google.com/go/storage v1.21.0
+	cloud.google.com/go/storage v1.22.0
```

Comment:
Just checking what motivated this bump?

Reply:
no longer bumped with the separate vendor PR